pkg/invariantlibrary/allowedbackenddisruption/matches_test.go

```diff
@@ -274,13 +275,14 @@ func TestDisruptionDataFileParsing(t *testing.T) {
 			foundMetalOVN = true
 		}
 	}
-
+	currentRelease = currentReleaseFromMap(releasesInQueryResults)
 	assert.Greater(t, dataOver100Runs, 5,
 		"expected at least 5 entries in query_results.json to have over 100 runs")
 	assert.True(t, foundAWSOVN, "no aws ovn job data in query_results.json")
 	assert.True(t, foundGCPOVN, "no gcp ovn job data in query_results.json")
 	assert.True(t, foundAzureOVN, "no azure ovn job data in query_results.json")
 	assert.True(t, foundMetalOVN, "no metal ovn job data in query_results.json")
+	// TO-DO update the assert to 2 when 4.15 branches, releaseInQueryResults will contain 2 releases after 4.15
```

Comment:
We will actually see the two releases in 4.14, we can look into a manual update of the file in origin when the ci-tools code merges.

Reply:
we won't see 2 releases in 4.14 as 4.13 doesn't have "MASTERNODEUPDATED" column, until we have 4.15 data we should be fine with assert checking for 1 release

Reply:
We will see 4.13 data for the non upgrade jobs.  You can verify by running the ci-tools update locally.